src/reportengine/configparser.py

```diff
@@ -132,6 +133,75 @@ def f_(self, val, *args, **kwargs):
 
     return f_
 
+def parse_record_(self, val):
```

Comment:
this identity function? I add parse functions for the lockfile keys i.e `parse_<key>_recorded_spec_` which are simply identity parsing functions because I was finding without this it was breaking and I wasn't sure why. I'll try and provide an example. Perhaps there is a better fix

Reply:
ok so I think it might just have been because I was make "foo" the name of one of the defaults for "foo" which in hindsight was really dumb. I think we can just take it out...